### DISCUSSION TITLE
App isn't syncing in background

### DISCUSSION THREAD

**Discussion by NikunjK8** [ReadYouApp/ReadYou#764] [Q&A]
I have battery optimisation disabled, background restart turned on and the app is locked in recent apps. But still the app isn't syncing in background.

These are the settings:
![IMG_20240613_114207](https://github.com/Ashinch/ReadYou/assets/99175083/7f273da3-1822-469a-9840-0972c2eb0793)

Tell me if I'm doing something wrong

**Comment by Liorst4** [comment]
Happens to me also.
I have the F-Droid version of the app.
Tried turn off battery optimization, but it didn't work.
Tried to "restart" the app using "force quit" in the settings app, do I need to restart the whole device for it to take effect?
